core/src/main/java/org/apache/accumulo/fate/ZooStore.java

```diff
@@ -174,7 +174,8 @@ public long reserve() {
 
           try {
             TStatus status = TStatus.valueOf(new String(zk.getData(path + "/" + txdir), UTF_8));
-            if (status == TStatus.IN_PROGRESS || status == TStatus.FAILED_IN_PROGRESS) {
+            if (status.equals(TStatus.SUBMITTED) || status.equals(TStatus.IN_PROGRESS)
+                || status.equals(TStatus.FAILED_IN_PROGRESS)) {
```

Comment:
```suggestion
            if (status == TStatus.SUBMITTED || status == TStatus.IN_PROGRESS
                || status == TStatus.FAILED_IN_PROGRESS) {
```

Reply:
Addressed in 46a503ace773ab8f3c1f35a91691c78014e43f9b